save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/utils/FileUtils.kt

```diff
@@ -26,6 +28,41 @@ expect fun Path.markAsExecutable()
  */
 expect fun ByteArray.writeToFile(file: Path, mustCreate: Boolean = true)
 
+/**
+ * Unzip [Path] if it is .zip file.
+ *
+ * @param markAsExecutables if true, marks all files as executables (true by default)
+ */
+fun Path.unzipIfRequired(markAsExecutables: Boolean = true) {
+    // FixMe: for now support only .zip files
```

Comment:
I guess `fixme` can be removed since the method has name `unzip***`

Reply:
Removed this todo